### ISSUE TITLE
NullPointerException: Attempt to invoke virtual method 'boolean org.wordpress.android.ui.reader.viewmodels.ReaderPostDe...

### ISSUE THREAD

**Issue by AliSoftware** [wordpress-mobile/WordPress-Android#15070]
**Sentry Url**: https://sentry.io/share/issue/43f418072d9d4e2587273d757d3ffee7/
**User Count**: 154
**Count**: 182
**First Release**: 17.7-rc-2
**First Seen**: 2021-07-08T06:23:06.957000Z
**Last Seen**: 2021-07-26T11:27:11Z
**24 Hours**: 17
**30 Days**: 181

```
NullPointerException: Attempt to invoke virtual method 'boolean org.wordpress.android.ui.reader.viewmodels.ReaderPostDetailViewModel$UiState.getErrorVisible()' on a null object reference
     at org.wordpress.android.ui.reader.ReaderPostDetailFragment$initViewModel$1.onChanged(ReaderPostDetailFragment.kt:442)
     at org.wordpress.android.ui.reader.ReaderPostDetailFragment$initViewModel$1.onChanged(ReaderPostDetailFragment.kt:132)
     at org.wordpress.android.ui.reader.viewmodels.ReaderPostDetailViewModel.updateRelatedPostsUiState(ReaderPostDetailViewModel.kt:480)
     at org.wordpress.android.ui.reader.viewmodels.ReaderPostDetailViewModel.access$buildLikersUiState(ReaderPostDetailViewModel.kt:77)
     at org.wordpress.android.ui.reader.viewmodels.ReaderPostDetailViewModel.access$updateRelatedPostsUiState(ReaderPostDetailViewModel.kt:77)
     at org.wordpress.android.ui.reader.viewmodels.ReaderPostDetailViewModel$onRelatedPostsRequested$1.invokeSuspend(ReaderPostDetailViewModel.kt:401)
...
(10 additional frame(s) were not displayed)
```

**Comment by AmandaRiu** [comment]
Not sure how `uiState` would be null, but that looks to be what's happening. Pinged on p1627334838469100-slack-C02QANACA

**Comment by AmandaRiu** [comment]
Fixed in #15083. Closing ticket.
